README.jp.md

```diff
@@ -0,0 +1,803 @@
+_Only the original [README](README.md) is guaranteed to be up-to-date._
+
+# scrcpy (v1.17)
+
+このアプリケーションはUSB(もしくは[TCP/IP経由][article-tcpip])で接続された
+Androidデバイスの表示と制御を提供します。このアプリケーションは  _root_ でのアクセス
+を必要としません。
+このアプリケーションは　_GNU/Linux_, _Windows_ そして _macOS_ 上で動作します。
+
+![screenshot](assets/screenshot-debian-600.jpg)
+
+以下に焦点を当てています:
+
+ - **軽量** (ネイティブ、デバイス画面表示のみ)
+ - **パフォーマンス**　(30~60fps)
+ - **クオリティ**　(1920x1080以上)
+ - **低遅延**　([35~70ms][lowlatency])
+ - **短い起動時間**　(初回画像を1秒以内に表示)
+ - **非侵入型**　(デバイスに何もインストールされない)
```

Comment:
It did install something on the device, but it will clean up on exit. Hence the "nothing is **left** installed".